gtoolkit/run.sh

```diff
@@ -0,0 +1,224 @@
+  local url=https://api.github.com/repos/feenkcom/gtoolkit/releases/latest
+  local json=$(curl -sL $url)
+
+  # Find the tag name line in the JSON
+  local versionLine=$(echo "$json" | grep tag_name)
+
+  # Scrape the value (adapted from https://stackoverflow.com/a/19394523)
+  local version=$(echo "$versionLine" | awk -F ': ' '/tag_name/ {gsub("\",?","");print $2}')
+
+  echo "$version"
+}
+
+gtoolkit::architecture() {
+  local gt_architecture
+
+  case "$(hardware_platform)" in
+    "arm64")
+      gt_architecture="aarch64"
+      ;;
+    "x86_64")
+      gt_architecture="x86_64"
+      ;;
+  esac
+
+  echo "$gt_architecture"
+}
+
+gtoolkit::archive_basename() {
+  local gt_architecture=$(gtoolkit::architecture)
+  local gt_platform
+  local gt_release=$(gtoolkit::latest_release_version)
+
+  if is_linux_build; then
+    gt_platform="Linux"
+  elif is_windows_build; then
+    if [[ "$gt_architecture" == "aarch64" ]]; then
+      print_error_and_exit "unsupported build platform '$(uname -s)'."
+    fi
+    gt_platform="Windows"
+  elif is_mac_build; then
+    gt_platform="MacOS"
+  else
+    print_error_and_exit "unsupported build platform '$(uname -s)'."
+  fi
+
+  echo "GlamorousToolkit-${gt_platform}-${gt_architecture}-${gt_release}"
+}
+
+gtoolkit::archive_url() {
+  local gt_release=$(gtoolkit::latest_release_version)
+  local url_base="https://github.com/feenkcom/gtoolkit/releases/download"
+
+  echo "${url_base}/${gt_release}/$(gtoolkit::archive_basename).zip"
+}
+
+gtoolkit::vm_path() {
+  local result
+
+  if is_linux_build; then
+    result="bin/GlamorousToolkit-cli"
+  elif is_windows_build; then
+    result="bin/GlamorousToolkit-cli.exe"
+  elif is_mac_build; then
+    result="GlamorousToolkit.app/Contents/MacOS/GlamorousToolkit-cli"
+  else
+    print_error_and_exit "unsupported build platform '$(uname -s)'."
+  fi
+
+  echo "${result}"
+}
+
+################################################################################
+# Download and move vm if necessary.
+# Globals:
+#   SMALLTALK_CI_VM
+################################################################################
+
+gtoolkit::prepare_vm() {
+  local vm_path="$(gtoolkit::vm_path)"
+
+  # Skip in case vm is already set up
+  if is_file "${SMALLTALK_CI_VM}"; then
+    return 0
+  fi
+
+  export SMALLTALK_CI_VM="${SMALLTALK_CI_BUILD}/${vm_path}"
+
+  chmod +x "${SMALLTALK_CI_VM}"
+
+  if ! is_file "${SMALLTALK_CI_VM}"; then
+    print_error_and_exit "Unable to set vm up at '${SMALLTALK_CI_VM}'."
+  fi
+}
+
+################################################################################
+# Download GT (image and VM) if necessary and copy it to build folder.
+# Globals:
+#   SMALLTALK_CI_BUILD
+#   SMALLTALK_CI_CACHE
+# Arguments:
+#   smalltalk_name
+################################################################################
+
+gtoolkit::prepare_gt() {
+  local smalltalk_name=$1 #Ignore currently because we are only supporting `release`
```

Comment:
Is the plan to eventually support something like `GToolkit-0.8.2593`?

Reply:
Yes, but since GT is still in beta and often releases several several times a day, it seemed easiest to get started by supporting the latest release.